src/ontology/omo-edit.owl

```diff
@@ -1498,7 +1505,10 @@ No imports</obo:IAO_0000115>
     <owl:NamedIndividual rdf:about="http://purl.obolibrary.org/obo/IAO_0000227">
         <rdf:type rdf:resource="http://purl.obolibrary.org/obo/IAO_0000225"/>
         <obo:IAO_0000111 xml:lang="en">terms merged</obo:IAO_0000111>
-        <obo:IAO_0000116 xml:lang="en">An editor note should explain what were the merged terms and the reason for the merge.</obo:IAO_0000116>
+        <obo:IAO_0000115 xml:lang="en">The term has been combined with one or more other terms to create a more encompassing (merged) term.</obo:IAO_0000115>
+        <obo:IAO_0000116 xml:lang="en">The deprecated term should reference the merged term with OIO:replaced_by. The replacement term should reference the original terms as sources using RO:derives_from. An editors note should explain what were the merged terms and the reason for the merge.</obo:IAO_0000116>
```

Comment:
```suggestion
        <obo:IAO_0000116 xml:lang="en">The deprecated term should reference the merged term with OIO:replaced_by. The replacement term should reference the original terms as sources using RO:derives_from. An editor note should explain what were the merged terms and the reason for the merge.</obo:IAO_0000116>
```

I suggest to be more coherent with the wording: this ontology contains "editor note" but "editors ontology module". (SKOS calls it "editorialNote")